src/Common/Platform.hpp

```diff
@@ -42,6 +42,9 @@
 #elif defined (_M_ARM64) || defined(__aarch64__)
 #   define XR_ARCHITECTURE_ARM64
 #   define _XRAY_ARCHITECTURE_MARKER "ARM 64-bit"
+#elif (defined(__riscv) || defined(__riscv__))
```

Comment:
Just a small code style correction
```suggestion
#elif defined(__riscv) || defined(__riscv__)
```